CHANGELOG

```diff
@@ -1,7 +1,9 @@
-Unreleased
+6.5.0
```

Comment:
next version is going to be 7.0.0 since some of the other things that have been merged are breaking